client/src/components/ideas/IdeaDetails.vue

```diff
@@ -112,6 +112,10 @@ export default {
       ideaReviews: [],
       ideaTypeCode: '0',
       upHere: '-1',
+      // Constants
+      PUBLIC_IDEA: 0,
+      PRIVATE_IDEA: 1,
+      COMMERCIAL_IDEA: 2,
```

Comment:
hmm I don't think these are needed, considering that in idea details we are just dealing with showing which type the idea is.